spec/controllers/projects/results_controller_spec.rb

```diff
@@ -32,134 +31,23 @@
-        create :query, result: old_result_not_used_for_comparison, statement: 'insert into users'
-
-        old_result_used_for_comparison = create :result,
-          project: project,
-          tag: 'pull-request-2',
-          example_name: 'user change user version 7 uses the correct timestamp'
-
-        create :query, result: old_result_used_for_comparison, statement: 'select * from users'
-        create :query, result: old_result_used_for_comparison, statement: 'delete * from users'
-
-        expected_result = [{
-          "example_name" => "user change user version 7 uses the correct timestamp",
-          "example_location" => "spec/integration/users/user_spec.rb",
-          "queries_that_got_added"=>["select * from teams where id in (?)"],
-          "queries_that_got_removed"=>["delete * from users"],
-        }]
-
-        expect(subject).to have_http_status :success
-        expect(body_json).to eq "results" => expected_result
-      end
-
-      context 'where the same query is used multiple times' do
-        it 'mentions the query twice' do
-          old_result = create :result,
-            project: project,
-            tag: 'pull-request-1',
-            example_name: 'user change user version 7 uses the correct timestamp'
-
-          create :query, result: old_result, statement: 'select * from users'
-          create :query, result: old_result, statement: 'select * from users'
-          create :query, result: old_result, statement: 'select * from users'
-          create :query, result: old_result, statement: 'select * from teams where id in (?)'
-
-          expected_result = [{
-            "example_name" => "user change user version 7 uses the correct timestamp",
-            "example_location" => "spec/integration/users/user_spec.rb",
-            "queries_that_got_added"=>[
-            ],
-            "queries_that_got_removed"=>[
-              'select * from users',
-              'select * from users',
-            ],
-          }]
-
-          expect(subject).to have_http_status :success
-          expect(body_json).to eq "results" => expected_result
-        end
-      end
-    end
-
-    describe 'without project no access' do
-      it do
-        expect { post(:create, params) }.to raise_error ActiveRecord::RecordNotFound
-      end
-    end
-  end
-
-  describe '.index' do
-    subject { get :index, project_id: project.id }
-
-    xit do
-      result = create :result
-      create :query, result: result
-      create :query, result: result
-
-      expect(subject).to have_http_status :success
-      expect(body_json).to eq(
-        "Users change reputation version 1 uses the correct timestamp" => [
-          {
-            "statement"=>"SELECT users.* FROM users WHERE id = 1"
-          },
-          {
-            "statement"=>"SELECT users.* FROM users WHERE id = 2"
-          }
-        ]
-      )
+    it 'without a project' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.